### PR TITLE
Added service to update an entity's ID

### PR DESCRIPTION
**Pull request by carlos-sarmiento**
## Description
Add a new service to update an entity's Entity ID.

## Motivation and Context

I run a rather large and sprawling Home Assistant instance. Sometimes I add an integration that creates 20+ entities. I don't like the way HA automatically builds the entity_id for new devices/entites and so have to manually go and rename them. The idea behind this is that, instead of having to manually rename every entity, or shutdown home assistant and mess with the entity_registry file in `.storage` I can do a bunch of service calls and get the result I want.

## How has this been tested?

Used on my own instance to rename entities. Service call was simple enough

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by carlos-sarmiento** [comment]
PR is missing the change to the entities documentation page. Wanted to make sure the change is welcomed before updating that and creating the screenshot

File: custom_components/spook/ectoplasms/homeassistant/services/update_entity_id.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    schema = {vol.Required("entity_id"): cv.entity_id, vol.Required("new_entity_id"): cv.entity_id}
+
+    async def async_handle_service(self, call: ServiceCall) -> None:
+        """Handle the service call."""
+        entity_registry = er.async_get(self.hass)
+        entity_id = call.data["entity_id"]
+        new_entity_id = call.data["new_entity_id"]
+        entity_registry.async_update_entity(
+            entity_id=entity_id,
+            new_entity_id=new_entity_id,
```
Comment (frenck) [inline custom_components/spook/ectoplasms/homeassistant/services/update_entity_id.py:31]:
```suggestion
        entity_registry.async_update_entity(
            entity_id=call.data["entity_id"],
            new_entity_id=call.data["new_entity_id"],
```

File: custom_components/spook/services.yaml [hunk 2]
```diff
@@ -224,6 +224,25 @@ homeassistant_enable_device:
...
+  description: >-
+    Updates an entity's ID on the fly.
+  fields:
+    entity_id:
+      name: Entity
+      description: The entity/entities to update.
+      required: true
+      selector:
+        entity:
+          multiple: false
```
Comment (frenck) [inline custom_components/spook/services.yaml:238]: The service doesn't seem to support multiple?

```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]
Fine by me!

Some small comments, and indeed: Docs :)

../Frenck

File: custom_components/spook/services.yaml (hunk 2 above)
Comment (carlos-sarmiento) [reply custom_components/spook/services.yaml]: Done

File: custom_components/spook/ectoplasms/homeassistant/services/update_entity_id.py (hunk 1 above)
Comment (carlos-sarmiento) [reply custom_components/spook/ectoplasms/homeassistant/services/update_entity_id.py]: Done

**Comment by carlos-sarmiento** [comment]
Updated everything. Should be ready to merge

**Review by frenck — APPROVED** [review]
Nice! Thanks, @carlos-sarmiento 👍 

../Frenck

**Pull request merged**